### PR TITLE
Add XPI file for Firefox extension

### PR DESCRIPTION
**Pull request by pranavmk98**

### CODE REVIEW

File: src/manifest.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "name": "CMUnits",
   "description": "Extends CMU Student Information Online interface with more data",
   "author": "Marat Valiev",
-  "version": "0.0.1.4",
```
Comment (user2589) [inline src/manifest.json]: nothing has changed in the source code. Please revert

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+Privacy: we do not store any data
+----
+
+Feel free to check the [source code](https://github.com/cmu-student-government/cmunit/tree/master/src)
+
+We appreciate your
+    [feedback](https://docs.google.com/forms/d/e/1FAIpQLSfIXZzPUekxeUMeR0l27fnTCPI_d5FEBOXGpMR4h2riG_FpqQ/viewform)!
+
+For Firefox users: The XPI file to install the extension can be found [here](https://raw.githubusercontent.com/cmu-student-government/cmunit/master/zip/cmunits.xpi)!
```
Comment (user2589) [inline docs/index.md]: Please replace the link with https://cmu-student-government.github.io/cmunit/cmunit.xpi

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+- No data is presented for a course if it is a new course, no data exists, or there were less than five responses the last time it was taught.
+
+Privacy: we do not store any data
+----
+
+Feel free to check the [source code](https://github.com/cmu-student-government/cmunit/tree/master/src)
+
+We appreciate your
+    [feedback](https://docs.google.com/forms/d/e/1FAIpQLSfIXZzPUekxeUMeR0l27fnTCPI_d5FEBOXGpMR4h2riG_FpqQ/viewform)!
+
```
Comment (user2589) [inline docs/index.md]: Please add Screenshots section, move pics/ under docs/ and insert pictures in the documents.

File: scripts/create_xpi.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+# Assuming this is run from repo root
+zip -jr zip/cmunits.xpi src
```
Comment (user2589) [inline scripts/create_xpi.sh]: Please change to:

	test $$(git config user.name) || git config user.name "Build bot (via TravisCI)"
	test $$(git config user.email) || git config user.email "build-bot@travis"
        zip -jr docs/cmunit.xpi src
        git add docs/cmunit.xpi
        git commit -m "build xpi extension"
        git push https://${GH_TOKEN}@github.com/cmu-student-government/cmunit.git master

This is will be a lot easier to use from CI

**Comment by user2589** [comment]
Thank you for submitting! Please make few final changes:
- please remove xpi file from the pull request - it should be built automatically by CI
- please revert move of compile_data.py. It relies on relative paths, and will break if moved without update.
- see inline comments in files 

Once done, I'll setup CI to automatically publish the file

**Comment by pranavmk98** [comment]
Incorporated all the changes suggested!

**Pull request merged**